vhdbuilder/packer/install-dependencies.sh

```diff
@@ -547,12 +543,16 @@ PATCHED_HYPERKUBE_IMAGES="
 1.18.8
 1.18.8-hotfix.20200924
 1.18.10
+1.18.10-hotfix.20210118
 1.18.14
+1.18.14-hotfix.20210118
 1.19.0
```

Comment:
i'd rather we remove 1.19.0 as it's been a bit old for 1.19

Reply:
removed a duplicate 1.18.8. Will see how build goes